rllib/agents/dqn/r2d2.py

```diff
@@ -32,8 +32,10 @@
 
         # === Replay buffer ===
         "replay_buffer_config": {
-            "_enable_replay_buffer_api": True,
             "type": "MultiAgentReplayBuffer",
+            # Specify prioritized replay by supplying a buffer type that supports
+            # prioritization
+            "prioritized_replay": DEPRECATED_VALUE,
```

Comment:
Is this needed here? It seems R2D2 didn't need this previously.

Reply:
PER was not used in R2D2 by default, but `prioritized_replay` it was a configuration option and is not being deprecated.